recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ package:
 
 source:
   git_url: https://github.com/openmm/{{ name }}.git
-  git_rev: 50d4b56b36d924c27650e03a4e98ab2d68a433b8  # current head of 7.5_branch
+  git_rev: a9cfd7fb9343e21c3dbb76e377c721328830a3ee  # current head of 7.5_branch
```

Comment:
Is the plan to encode the git rev here and _then_ cut a 7.5.0 release on GitHub for this exact git hash?
Wouldn't it be better to cut the GitHub release first, then use that release version for `git_rev`?

Reply:
This is a release candidate.  We won't create a github release until it's final.

Reply:
The GitHub release is not necessary, but it'd be preferred to at least have the git tag so it's easier to check which commit corresponded to each RC. Otherwise we'd only have the feedstock git history?

Reply:
Yeah, now that we've automated builds, I don't see any compelling reason to not cut the GitHub release first (for the release candidates, marked as pre-releases) and then use that git release tag in the recipe.